calico-enterprise/multicluster/set-up-multi-cluster-management/helm-install/create-a-management-cluster-helm.mdx

```diff
@@ -156,11 +155,52 @@ To install a {{prodname}} [management](create-a-management-cluster-helm#value) c
   watch kubectl get tigerastatus
   ```
 
+
+#### Update the ManagementCluster address.
+
+1. Export the service port number
+
+  ```bash
+  export EXT_NODE_PORT=<your-load-balancer-external-node-port>
+  ```
+
+  Export the public address or host of the management cluster, in this case the load-balancer's external IP (Ex. "example.com:1234" or "10.0.0.10:1234".)
+
+  ```bash
+  export MANAGEMENT_CLUSTER_ADDR=<your-load-balancer-external-addr>:$EXT_NODE_PORT
+  ```
+
+  Replace the `address` field in the ManagementCluster resource.
+    
+  ```bash
+  kubectl patch managementcluster tigera-secure --type merge -p "{\"spec\":{\"address\":\"${MANAGEMENT_CLUSTER_ADDR}\"}}"
+  ``` 
+
 </TabItem>
 <TabItem label="LoadBalancer" value="LoadBalancer">
 
 To install a {{prodname}} [management](create-a-management-cluster-helm#value) cluster with Helm, using a LoadBalancer service:
 
+#### Set-up an EKS LoadBalancer.
+
+  You must meet the requirements defined in [create a network load balancer for AWS](https://docs.aws.amazon.com/eks/latest/userguide/network-load-balancing.html)
```

Comment:
This section assumes that the user is on EKS.

Would suggest something more generic:
```suggestion
#### Meet cloud provider requirements.

You must ensure that you have met the requirements for your cloud provider to provision a load balancer in your environment.

For example, if you are using EKS, you must meet the requirements defined in [create a network load balancer for AWS](https://docs.aws.amazon.com/eks/latest/userguide/network-load-balancing.html)
```